specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1770,6 +1887,130 @@
                 "sku"
             ]
         },
+        "TopologyList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/TopologyResource"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "TopologyResource": {
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "readOnly": true,
+                    "$ref": "#/definitions/TopologyResourceProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/Location"
+                }
+            ]
+        },
+        "TopologyResourceProperties": {
+            "type": "object",
+            "properties": {
+                "calculatedDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "The UTC time on which the topology was calculated"
+                },
+                "resources": {
+                    "type": "array",
+                    "readOnly": true,
+                    "description": "Azure resources which are part of this topology resource",
+                    "items": {
+                        "$ref": "#/definitions/TopologySingleResource"
+                    }
+                }
+            }
+        },
+        "TopologySingleResource": {
+            "type": "object",
+            "properties": {
+                "resourceId": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "Azure resource id"
+                },
+                "severity": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "The security severity of the resource"
+                },
+                "recommendationsExist": {
+                    "type": "boolean",
+                    "readOnly": true,
+                    "description": "Indicates if the resource has security recommendations"
+                },
+                "internetFacing": {

```

Comment:
bools are generally not recommended, they are less descriptive and have removes any scope of future expansion in case you need more states to be reflected through that property.

Reply:
Agree. In the UI we actually show this as "Network Zones", changed to networkZones here as well (InternetFacing is one of the options).